ScalaApp/scala_settings.py

```diff
@@ -19,8 +19,8 @@
 KITCHEN_USE_END_TIME = time(19, 30)
 
 # Balance bottom limit
-MINIMUM_BALANCE_FOR_DINING_SIGN_UP = Decimal('-2.00') - KITCHEN_COST
-MINIMUM_BALANCE_FOR_DINING_SLOT_CLAIM = Decimal('-2.00') - KITCHEN_COST
+MINIMUM_BALANCE_FOR_DINING_SIGN_UP = Decimal('-2.00') + KITCHEN_COST
+MINIMUM_BALANCE_FOR_DINING_SLOT_CLAIM = Decimal('-2.00') + KITCHEN_COST
```

Comment:
:smile: Wessel noticed it but we forgot to tell you this

Reply:
And you didn't think to fix it yourself... boooo :P